### ISSUE TITLE
Direct Touch / 2D app support on HorizonOS/Quest

### ISSUE THREAD

**Issue by Gatorzgaming** [Igalia/wolvic#1454]
Currently (outside of side loading 2D android browsers) the built in quest browser is the only direct touch support web browser for quest headsets; I would love to have an option to put Wolvic into 2D App/ Direct Touch mode.

A way to click to transition from the 6DOF view to a 2D app

Alteratives: 
Side loading an android Browser :/ 

I apologize if this is a redundant issue; I was unable to find an open issue about this.

**Comment by DimkaAdam** [comment]
Did you find a solution?

**Comment by Gatorzgaming** [comment]
No, this was a feature request. Which is why its labeled as "enhancement"

**Comment by nerdbloke** [comment]
This was released at Meta Connect 2024 in September:
https://developers.meta.com/horizon/develop/spatial-sdk
